File: ch02_neutronDiffusion/matlab/sketch_triangle.m
clear
close all

LW = 2;
tri = [...
    .7,.5;
    .2,.5;
    0,-.1;
    .7,.5];
    

figure
plot(tri(:,1),tri(:,2),'-o','Color','k','LineWidth',LW,'MarkerFaceColor','k')
axis equal

set(gca,'Visible','off')
print('../figs/sketch_triangle.eps','-depsc2')
close(gcf)

File: ch02_neutronDiffusion/python/sketch_triangle.py
import numpy as np
import matplotlib.pyplot as plt

LW = 2
FN = 'Times New Roman'
FS = 12
plt.rc('lines', lw=LW)
plt.rc('mathtext', fontset='stix') # not explicitly Times New Roman but a good clone
plt.rc('font', family=FN, size=FS)

tri = np.array([
    [0.7, 0.5],
    [0.2, 0.5],
    [0.0, -0.1],
    [0.7, 0.5]])

plt.figure()
plt.plot(tri[:,0], tri[:,1], '-ko')
plt.axis('equal')
plt.axis('off')
plt.tight_layout()
plt.savefig('../figs/sketch_triangle.pdf', bbox_inches='tight', pad_inches=0)
plt.close()
